io.github.imamhs.Shopne.yml

```diff
@@ -0,0 +1,85 @@
+app-id: io.github.imamhs.Shopne
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: shopne
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --filesystem=host:ro
+  - --filesystem=xdg-config/shopne:create
+  - --device=all
+modules:
+  - name: shopne
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 Shopne /app/games/shopne/Shopne
+      - install -Dm755 roms.py /app/games/shopne/roms.py
+      - install -Dm755 roms_directory_selection_dialog.py /app/games/shopne/roms_directory_selection_dialog.py
+      - install -Dm755 engine.py /app/games/shopne/engine.py
+      - install -Dm755 gngeo/gngeo_bin /app/games/shopne/gngeo/gngeo_bin
+      - cp gngeo/gngeo_data.zip /app/games/shopne/gngeo
+      - cp gngeo/COPYING /app/games/shopne/gngeo
+      - cp -r data/ /app/games/shopne/
+      - mkdir -p /app/bin/
+      - install -Dm755 shopne_flatpak /app/bin/shopne
+      - cp License.txt /app/games/shopne
+      - cp ReadMe.md /app/games/shopne
+      - install -Dm755 io.github.imamhs.Shopne.metainfo.xml /app/share/metainfo/io.github.imamhs.Shopne.metainfo.xml
+      - install -Dm755 data/icon.png /app/share/icons/hicolor/128x128/apps/io.github.imamhs.Shopne.png
+      - install -Dm755 io.github.imamhs.Shopne.desktop /app/share/applications/io.github.imamhs.Shopne.desktop
+    sources:
+      - type: dir
+        path: .
+  - name: libsdl
+    buildsystem: simple
+    build-commands:
+      - ./configure --prefix=/app
+      - sed -i '168s/.*/SDL_X11_SYM(int,_XData32,(Display *dpy,register _Xconst long *data,unsigned len),(dpy,data,len),return)/' ./src/video/x11/SDL_x11sym.h
+      - make
+      - make install
+    sources:
+      - type: archive
+        url: https://www.libsdl.org/release/SDL-1.2.15.tar.gz
+        sha256: d6d316a793e5e348155f0dd93b979798933fb98aa1edebcc108829d6474aad00
```

Comment:
use the shared module https://github.com/flathub/shared-modules/blob/master/SDL/SDL-1.2.15.json